roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -201,11 +201,7 @@ spec:
             - name: CALICO_IPV4POOL_IPIP
               value: "{{ calico_ipv4pool_ipip }}"
             - name: FELIX_IPV6SUPPORT
-{% if enable_dual_stack_networks %}
-              value: "true"
-{% else %}
-              value: "false"
-{% endif %}
+              value: "{{ enable_dual_stack_networks | default(false) }}"
```

Comment:
squash with the commit that introduced the change

Reply:
Amended in previous commit